projects/plugins/crm/sass/_ZeroBSCRM.adminpages.scss

```diff
@@ -572,8 +572,16 @@
     color:white;
 	margin-right:0.4em;
 }
-.learn{
-	margin-left: 10px !important;
+
+#learn {
```

Comment:
It's better to use CSS classes instead: `.learn.button` That way it will reusable multiple times on the same page.

Reply:
I think the issue you had is that other classes override yours. The trick here is to be more specific with the classes: `div.ui.learn.button`

Reply:
Thanks for the tip!